files/zh-cn/.markdownlint.jsonc

```diff
@@ -63,6 +63,13 @@
         "searchPattern": "/\\]\\(https?:\\/\\/localhost:\\d+\\//g",
         "replace": "](/",
         "searchScope": "text"
+      },
+      {
+        "name": "force-pronoun",
+        "message": "Don't use '您'",
```

Comment:
What about

```
Consider using '你' instead of '您'
```

I'm not sure if message here should instruct user what to replace 🤔

Reply:
Should we also remove the replace field?

Reply:
> Should we also remove the replace field?

Make sense 👍